beast/fitting/tests/test_fit_grid.py

```diff
@@ -136,7 +38,7 @@ def test_fit_grid():
     basefilters = ["F275W", "F336W", "F475W", "F814W", "F110W", "F160W"]
     obs_colnames = [f.lower() + "_rate" for f in basefilters]
 
-    obsdata = get_obscat(obs_fname, filters, obs_colnames, vega_fname=vega_fname)
+    obsdata = Observations(obs_fname, filters, obs_colnames, vega_fname=vega_fname)
     # output files
     stats_fname = "/tmp/beast_example_phat_stats.fits"
     pdf1d_fname = "/tmp/beast_example_phat_pdf1d.fits"
```

Comment:
Do we want to add pdf2d_fname as well?

Reply:
We should add checking the pdf2d results as well.  But that could be done in a separate PR.  Probably better that way as long as an issue is created so we don't forgot.  Does this sound reasonable?

Reply:
I've created issue #566.